src/libs/Navigation/linkingConfig/CENTRAL_PANE_TO_RHP_MAPPING.ts

```diff
@@ -2,12 +2,40 @@ import type {CentralPaneName} from '@libs/Navigation/types';
 import SCREENS from '@src/SCREENS';
 
 const CENTRAL_PANE_TO_RHP_MAPPING: Partial<Record<CentralPaneName, string[]>> = {
-    [SCREENS.WORKSPACE.PROFILE]: [SCREENS.WORKSPACE.NAME, SCREENS.WORKSPACE.CURRENCY, SCREENS.WORKSPACE.DESCRIPTION, SCREENS.WORKSPACE.SHARE],
-    [SCREENS.WORKSPACE.REIMBURSE]: [SCREENS.WORKSPACE.RATE_AND_UNIT, SCREENS.WORKSPACE.RATE_AND_UNIT_RATE, SCREENS.WORKSPACE.RATE_AND_UNIT_UNIT],
-    [SCREENS.WORKSPACE.MEMBERS]: [SCREENS.WORKSPACE.INVITE, SCREENS.WORKSPACE.INVITE_MESSAGE, SCREENS.WORKSPACE.MEMBER_DETAILS, SCREENS.WORKSPACE.MEMBER_DETAILS_ROLE_SELECTION],
-    [SCREENS.WORKSPACE.WORKFLOWS]: [SCREENS.WORKSPACE.WORKFLOWS_APPROVER, SCREENS.WORKSPACE.WORKFLOWS_AUTO_REPORTING_FREQUENCY, SCREENS.WORKSPACE.WORKFLOWS_AUTO_REPORTING_MONTHLY_OFFSET],
-    [SCREENS.WORKSPACE.CATEGORIES]: [SCREENS.WORKSPACE.CATEGORY_CREATE, SCREENS.WORKSPACE.CATEGORY_SETTINGS, SCREENS.WORKSPACE.CATEGORIES_SETTINGS, SCREENS.WORKSPACE.CATEGORY_CREATE],
-    [SCREENS.WORKSPACE.TAGS]: [SCREENS.WORKSPACE.TAGS_SETTINGS, SCREENS.WORKSPACE.TAGS_EDIT],
+    [SCREENS.SETTINGS.PROFILE.ROOT]: [
+        SCREENS.SETTINGS.PROFILE.DISPLAY_NAME,
+        SCREENS.SETTINGS.PROFILE.CONTACT_METHODS,
+        SCREENS.SETTINGS.PROFILE.CONTACT_METHOD_DETAILS,
+        SCREENS.SETTINGS.PROFILE.NEW_CONTACT_METHOD,
+        SCREENS.SETTINGS.PROFILE.STATUS_CLEAR_AFTER,
+        SCREENS.SETTINGS.PROFILE.STATUS_CLEAR_AFTER_DATE,
+        SCREENS.SETTINGS.PROFILE.STATUS_CLEAR_AFTER_TIME,
+        SCREENS.SETTINGS.PROFILE.STATUS,
+        SCREENS.SETTINGS.PROFILE.PRONOUNS,
+        SCREENS.SETTINGS.PROFILE.TIMEZONE,
+        SCREENS.SETTINGS.PROFILE.TIMEZONE_SELECT,
+        SCREENS.SETTINGS.PROFILE.LEGAL_NAME,
+        SCREENS.SETTINGS.PROFILE.DATE_OF_BIRTH,
+        SCREENS.SETTINGS.PROFILE.ADDRESS,
+        SCREENS.SETTINGS.PROFILE.ADDRESS_COUNTRY,
+        SCREENS.SETTINGS.SHARE_CODE,
+    ],
+    [SCREENS.SETTINGS.PREFERENCES.ROOT]: [SCREENS.SETTINGS.PREFERENCES.PRIORITY_MODE, SCREENS.SETTINGS.PREFERENCES.LANGUAGE, SCREENS.SETTINGS.PREFERENCES.THEME],
+    [SCREENS.SETTINGS.WALLET.ROOT]: [
+        SCREENS.SETTINGS.WALLET.DOMAIN_CARD,
+        SCREENS.SETTINGS.WALLET.CARD_GET_PHYSICAL.NAME,
+        SCREENS.SETTINGS.WALLET.CARD_GET_PHYSICAL.PHONE,
+        SCREENS.SETTINGS.WALLET.CARD_GET_PHYSICAL.ADDRESS,
+        SCREENS.SETTINGS.WALLET.CARD_GET_PHYSICAL.CONFIRM,
+        SCREENS.SETTINGS.WALLET.TRANSFER_BALANCE,
+        SCREENS.SETTINGS.WALLET.CHOOSE_TRANSFER_ACCOUNT,
+        SCREENS.SETTINGS.WALLET.ENABLE_PAYMENTS,
+        SCREENS.SETTINGS.WALLET.CARD_ACTIVATE,
+        SCREENS.SETTINGS.WALLET.REPORT_VIRTUAL_CARD_FRAUD,
+        SCREENS.SETTINGS.WALLET.CARDS_DIGITAL_DETAILS_UPDATE_ADDRESS,
+    ],
+    [SCREENS.SETTINGS.SECURITY]: [SCREENS.SETTINGS.TWO_FACTOR_AUTH, SCREENS.SETTINGS.CLOSE],
+    [SCREENS.SETTINGS.ABOUT]: [SCREENS.SETTINGS.APP_DOWNLOAD_LINKS, SCREENS.SETTINGS.TROUBLESHOOT],
 };
 
```

Comment:
WORKSPACE SCREEN mappings removed? Central Pane gets changed when navigating to Edit Category or New Category Pages now.

Reply:
For me works it well 


https://github.com/Expensify/App/assets/24796318/be448288-069b-41f5-bacd-10e9a9b598bb

Reply:
I am able to recreate consistently in an incognito tab but not in another tab. I don't understand what is causing this.


https://github.com/Expensify/App/assets/102477862/16e5ad3e-59ce-48a6-9e91-2169cc87edc7

Reply:
We can also see the issue if we reload the page, the central pane changes to the report screen

https://github.com/Expensify/App/assets/22219519/25084159-5f7a-4c2a-9c9e-76232215f277

Reply:
fixed

Reply:
This is still not fixed. I don't think this is due to any changes in this PR. This might be due to this [change here](https://github.com/Expensify/App/commit/71a24255d6075d3d45b13980a994caab51f6a504#r139773715).


https://github.com/Expensify/App/assets/102477862/5126b104-0cae-44f5-93b0-8ba3cdc15fd3

Reply:
I've noticed the same issues across all RHP pages in the workspace settings. Would a dedicated PR related to the ideal navbar best resolve this navigation issue?

cc @luacmartins 

https://github.com/Expensify/App/assets/24796318/bc9855e5-4a5a-4bb1-99ae-4af69f14676b

Reply:
Yea, if this is a generalized problem, let's fix it in a separate PR